pkg/controller/upgradeconfig/cluster_upgrader.go

```diff
@@ -255,7 +255,7 @@ func CommenceUpgrade(c client.Client, m maintenance.Maintenance, upgradeConfig *
 // Create the maintenance window for control plane
 func CreateControlPlaneMaintWindow(c client.Client, m maintenance.Maintenance, upgradeConfig *upgradev1alpha1.UpgradeConfig, reqLogger logr.Logger) (bool, error) {
 	endTime := time.Now().Add(90 * time.Minute)
```

Comment:
Not a commit specific to your PR changes, but, they made me think of it.
In the maintenance client you explicitly convert to UTC with "time.Now().UTC()" when generating the 'now' timestamp.
For consistency should we do the same when generating the endTime?
(It's a minor nit and yeah all our servers are on UTC)

Reply:
Nice catch @mrbarge 
Check 2c192d67cacd8bdf7202c439afbde1f97d1439a0